### PR TITLE
Pause on slow finding

### LINKED ISSUE #101: Lorsque l'on utilise le contrôle permettant plus de précision, arrêter la vidéo
### Before submitting an issue, make sure you have:
- [X] Read the [README](https://github.com/TSAP-Laval/acquisition-frontend/blob/master/README.md)
- [X] [Searched](https://github.com/TSAP-Laval/acquisition-frontend/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [X] Reviewed the sample code in [tests](https://github.com/TSAP-Laval/acquisition-frontend/tree/master/tests)

### Purpose of your issue?
- [X] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### Describe your issue

Lorsque l'on décide de faire une avance plus précise avec le 'slider' de droite, la vidéo ne se pause pas et donc il devient plus difficile d'être précis... Un peu comme le contrôle du temps.

### PR DESCRIPTION
**Pull request by Alexandrehebertvincent**
## Please follow the guide below

- Pull requests (PR) submitted without this template format will likely be **ignored**.
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your PR (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like.

---

### Before submitting a pull request, make sure you have:
- [x] Read the [README](https://github.com/TSAP-Laval/acquisition-frontend/blob/master/README.md)
- [x] [Searched](https://github.com/TSAP-Laval/acquisition-frontend/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [ ] Reviewed the sample code in [tests](https://github.com/TSAP-Laval/acquisition-frontend/tree/master/tests)

### Purpose of your pull request?
- [x] Bug fix (encountered problems/errors)
- [ ] Feature (request of a new functionality)
- [x] Enhancement
- [ ] Other

### If people needs to do some things, include them with their tasks as shown bellow:
1. The title and description of the task.
    - [ ] Alex D
    - [ ] Alex H
    - [ ] Mehdi
    - [ ] Laurent

---

### Complete and detailed description of your PR*

* If no description is provided, the PR will be ignored.

### Fixes #101 
Le vidéo se met en pause lorsque l'utilisateur utilise la recherche précise.

### CODE REVIEW

File: src/components/Uploader/Form.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export default class Form extends React.Component<ILayoutProps, ILayoutState> {
 
     public onSave() {
         if (!this.state.savedOnce) {
-            this.state.savedOnce = true;
+            this.setState({savedOnce: true});
             this.shouldComponentUpdate(this.state);
```
Comment (laurentlp) [inline src/components/Uploader/Form.tsx]: Peut être enlever dans le cas où setState() est utilisé

**Review by laurentlp — CHANGES_REQUESTED** [review]
Fais juste changer ça et c'est parfait ! @Alexandrehebertvincent

**Comment by laurentlp** [comment]
Ah ! Aussi oublie pas de linter ton code, sinon ça ne passera pas sur dev ! Fais juste installer tslint comme expliqué dans le readme front-end !

File: src/components/Uploader/Form.tsx (hunk 1 above)
Comment (Alexandrehebertvincent) [reply src/components/Uploader/Form.tsx]: J'enlève `this.setState({savedOnce: true});`?

Comment (laurentlp) [reply src/components/Uploader/Form.tsx]: Non `this.shouldComponentUpdate(this.state);`! De toute façon j'ai mis pas mal tous les packages à jour et ça va tout changer cette partie donc vas-y. La fonction `shouldComponentUpdate`servait à ''updater'' le _state_ du module manuellement...

File: src/actions/VideoPlayerActions.tsx [hunk 2]
```diff
@@ -1,17 +1,20 @@
 import dispatcher from "../dispatcher/dispatcher";
 
+// tslint:disable:object-literal-shorthand
```
Comment (laurentlp) [inline src/actions/VideoPlayerActions.tsx]: Ne disable pas le linter sans justifier la raison plz !

**Review by laurentlp — CHANGES_REQUESTED** [review]

File: src/actions/VideoPlayerActions.tsx (hunk 2 above)
Comment (Alexandrehebertvincent) [reply src/actions/VideoPlayerActions.tsx]: Allright!

Comment (laurentlp) [reply src/actions/VideoPlayerActions.tsx]: 👍

**Review by laurentlp — APPROVED** [review]

**Pull request merged**